### ISSUE TITLE
Generate scenario for each string?

### ISSUE THREAD

**Issue by ben-eb** [shoreditch-ops/artillery-plugin-fuzzer#3]
Currently the plugin selects strings at random from the list, which does not guarantee that any particular string is sent to the server, and could result in a few thousand runs where not all of the strings end up being sent. I'm not exactly sure how it should be implemented but I'd like to see a mode where this plugin could run all of the strings against an endpoint.

**Comment by hassy** [comment]
Good idea, cheers. It could be done with [a `loop`](https://artillery.io/docs/http-reference/#looping-through-an-array) if the plugin exported a list with all naughty strings.
